### ISSUE TITLE
[1.32.0] Unable to interact or assert with a sheet inside a modal

### ISSUE THREAD

**Issue by adgifs** [mobile-dev-inc/maestro#1503]
**Describe the bug**
I cannot assert or interact with my sheet - refer to the below screenshot. The sheet doesn't show up in the maestro hierarchy either, therefore I'm unable to find any of my accessibility identifiers. I have noticed that this only happens to a sheet that is inside a .fullScreenCover(). I have made sure that the views are accessible even with `accessibilityElement(children: .contains)` and with multiple assigned ids.

**To Reproduce**
Steps to reproduce the behavior, e.g.:

1. Start the simulator
2. Run maestro studio
3. The sheet view is not interactable or assertable even though it is visible

**Expected behavior**
I should be able to assert and interact with my sheet when it is presented.

**Screenshots**
Inside a .fullScreenCover(), which isn't working:
<img width="247" alt="image" src="https://github.com/mobile-dev-inc/maestro/assets/135635546/5fc63b47-6f2d-404d-816a-fb48577a731f">

The exact same view but outside a .fullScreenCover():
<img width="292" alt="image" src="https://github.com/mobile-dev-inc/maestro/assets/135635546/c80ca8df-9533-478e-ae03-3979732780b7">

**Environment information (please complete the following information):**

- Maestro version: 1.32.0
- Platform: iOS 16.4
- Framework: Native iOS
- Device model and OS version: iPhone 14 Pro
- Simulator, emulator or physical device: Simulator
- Host [e.g. Mac M1, Windows, Linux, etc]: Mac M2

**Bugreport output (if applicable)**
[2023-10-03_134153.zip](https://github.com/mobile-dev-inc/maestro/files/12792525/2023-10-03_134153.zip)

**Comment by Othman2001** [comment]
@adgifs  any updates on this issue, I am still facing it with react native  bottom sheet

**Comment by andreaSimonePorceddu** [comment]
Same for me, cannot interact in a modal (only for iOS)

**Comment by adgifs** [comment]
> @adgifs any updates on this issue, I am still facing it with react native bottom sheet

Nothing so far

**Comment by bartekpacia** [comment]
Hey, thanks y'all for creating this issue - unfortunately, it's not easily reproducible. We have a huge backlog, so it's important we can triage issues effectively.

Please provide [a minimal, reproducible example] that clearly explains how to reproduce the problem with the detailed, step-by-step instructions (that can be easily run by me or other maintainers).

[minimal, reproducible example]: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by LoopingStudio** [comment]
Describe the bug
I cannot assert or interact with my sheet - refer to the below screenshot. The sheet doesn't show up in the maestro hierarchy either, therefore I'm unable to find any of my accessibility identifiers. I have noticed that this only happens to a sheet that is inside a .fullScreenCover(). I have made sure that the views are accessible even with accessibilityElement(children: .contains) and with multiple assigned ids.

To Reproduce
Steps to reproduce the behavior, e.g.:

Start the simulator
Run maestro studio
The sheet view is not interactable or assertable even though it is visible
Expected behavior
I should be able to assert and interact with my sheet when it is presented.
